Packs/Zimperium/Classifiers/classifier-mapper-incoming-Zimperium.json

```diff
@@ -0,0 +1,92 @@
+{
+	"id": "Zimperium - mapper",
+	"name": "Zimperium - Incoming Mapper",
+	"type": "mapping-incoming",
+	"version": -1,
+	"fromversion": "6.0.0",
+	"description": "Maps incoming Zimperium incident fields.",
+	"defaultIncidentType": "",
```

Comment:
no default incident type?

Reply:
Not AFAIK, they can choose to use the `Zimperium event` incidenttype, but not sure we want it as default. @Arsenikr WDYT?